runtime/onert/backend/train/ops/PadLayer.cc

```diff
@@ -34,16 +34,20 @@ PadLayer::PadLayer() : cpu::ops::PadLayer(), _back_prop_input{nullptr}, _back_pr
 
 template <typename T> void PadLayer::depad()
 {
-  nnfw::cker::train::Depad<T>(_padData, _padRank, getShape(_back_prop_output),
+  assert(_pad->data_type() == onert::ir::DataType::INT32);
+  assert(_pad->buffer());
+  const auto pad_data = reinterpret_cast<const int32_t *>(_pad->buffer());
+  auto pad_rank = _pad->getShape().dim(0);
```

Comment:
(Optional)
How about adding an assertion before this like below?
```c++
  assert(_pad->getShape().rank() == 1);
```

Reply:
That's a good point. I'll fix it and another patch for also `padImpl()` in cker.

Reply:
@ragmani ah, it cannot. the `_pad`'s shape could be like `[4,2]`.